scripts/units/ANIfrog.js

```diff
@@ -247,18 +247,30 @@ frog.constructor = () => extend(MechUnit, {
 
 frog.defaultController = prov(() => extend(AIController, {}));
 
-Events.on(Trigger.update.getClass(), () => {
-    if(!Mathf.chanceDelta(0.0001) || !Vars.state.isPlaying()) return;
+var map_valids = [];
+var map_valids_usable = false:
```

Comment:
`false:`
the only error of the entire code, but thats probably because of mobile keyboard